src/dao/pages/DAOsListPage/DAOCard.tsx

```diff
@@ -13,10 +14,12 @@ export interface DAOCardProps {
 export function DAOCard(props: DAOCardProps) {
   const { name, type, accountId, logoUrl } = props;
   const navigate = useNavigate();
+  const daoAccountIdQueryResults = useContract(accountId);
```

Comment:
it should be `evmAddress` in the `useContract` param

Reply:
accountId is the EVMAddress for MultiSig but for FT and NFT DAOs it is AccountId. so with this PR I have not changed the name of the variable. I wull update it with my next PR, in which I am making changes in the FT and NFT DAOs.